src/openstudio_lib/library/OpenStudioPolicy.xml

```diff
@@ -1236,7 +1230,7 @@
     <rule IddField="Nominal Floor to Ceiling Height" Access="LOCKED"/>
   </POLICY>
   <POLICY IddObjectType="OS_DefaultConstructionSet">
-    <rule IddField="Name" Access="LOCKED"/>
+    <!--<rule IddField="Name" Access="LOCKED"/>-->
```

Comment:
What's this about?

Reply:
From another PR: https://github.com/openstudiocoalition/OpenStudioApplication/pull/107/files#diff-3f25226a4f407224991996e30426d766R1233